signac_dashboard/dashboard.py

```diff
@@ -548,10 +589,26 @@ def modify_query(**new_values):
         def page_not_found(error):
             return self._render_error(str(error))
 
+        @dashboard.login_manager.unauthorized_handler
+        def unauthorized_handler():
+            return self._render_error("Access token is required.")
+
+        @dashboard.app.route("/login")
+        def login():
+            provided_token = request.args.get("token")
+            if provided_token == self.config["ACCESS_TOKEN"]:
+                user = User(provided_token)
+                flask_login.login_user(user)
+                return redirect("/")
+
+            return self._render_error("Invalid token")
+
         @dashboard.app.route("/favicon.ico")
+        @flask_login.login_required
         def favicon():
             return url_for("static", filename="favicon.ico")
 
+        # These routes are protected within the LazyView utility class
```

Comment:
I wanted to point people to what you added in the `__call__` function of `LazyView`